### ISSUE TITLE
log line accumulation JSON logging 

### ISSUE THREAD

**Issue by odise** [gin-contrib/logger#78]
I just noticed that the `/json` endpoint seems to accumulating log messages in a strange (non-compliant JSON) way since version 0.3.0. Here is an example based on the code in the  `_example` directory. In a nutshell log lines will grow with each request.

```
go run main.go
[GIN-debug] [WARNING] Running in "debug" mode. Switch to "release" mode in production.
 - using env:   export GIN_MODE=release
 - using code:  gin.SetMode(gin.ReleaseMode)

[GIN-debug] GET    /pong                     --> main.main.func1 (2 handlers)
[GIN-debug] GET    /ping                     --> main.main.func2 (2 handlers)
[GIN-debug] GET    /skip                     --> main.main.func3 (2 handlers)
[GIN-debug] GET    /regexp1                  --> main.main.func4 (2 handlers)
[GIN-debug] GET    /regexp2                  --> main.main.func5 (2 handlers)
[GIN-debug] GET    /id                       --> main.main.func8 (3 handlers)
[GIN-debug] GET    /json                     --> main.main.func10 (2 handlers)
[GIN-debug] [WARNING] You trusted all proxies, this is NOT safe. We recommend you to set a value.
Please check https://pkg.go.dev/github.com/gin-gonic/gin#readme-don-t-trust-all-proxies for details.
[GIN-debug] Listening and serving HTTP on :8080
{"level":"info","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.013785,"user_agent":"curl/8.4.0","time":"2024-01-18T10:45:20+01:00","message":"Request"}
{"level":"info","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.013785,"user_agent":"curl/8.4.0","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.00572,"user_agent":"curl/8.4.0","time":"2024-01-18T10:45:22+01:00","message":"Request"}
{"level":"info","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.013785,"user_agent":"curl/8.4.0","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.00572,"user_agent":"curl/8.4.0","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.009236,"user_agent":"curl/8.4.0","time":"2024-01-18T10:45:24+01:00","message":"Request"}
{"level":"info","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.013785,"user_agent":"curl/8.4.0","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.00572,"user_agent":"curl/8.4.0","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.009236,"user_agent":"curl/8.4.0","status":200,"method":"GET","path":"/json","ip":"::1","latency":0.014365,"user_agent":"curl/8.4.0","time":"2024-01-18T10:45:27+01:00","message":"Request"}
```

Any idea whats wrong here?

**Comment by mgabeler-lee-6rs** [comment]
This is #68, to be fixed by #69

**Comment by mgabeler-lee-6rs** [comment]
This should be fixed in v1.1.0: https://github.com/gin-contrib/logger/pull/69#issuecomment-1972337492
